modules/cluster-logging-collector-log-forward-secret-cloudwatch.adoc

```diff
@@ -1,19 +1,25 @@
 // Module included in the following assemblies:
 //
-// * observability/logging/cluster-logging-external.adoc
+// * configuring/configuring-log-forwarding.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="cluster-logging-collector-log-forward-secret-cloudwatch_{context}"]
 = Creating a secret for AWS CloudWatch with an existing AWS role
-If you have an existing role for AWS, you can create a secret for AWS with STS using the `oc create secret --from-literal` command.
+
+Create a secret for {aws-first} {sts-first} from the configured {aws-short} IAM role by using the `oc create secret --from-literal` command.
```

Comment:
nit:    another example, but {aws-short} IAM is redundant and we could remove the "cloud" terms.  This entire section/document is about AWS, so it would be much cleaner to just mention "IAM role", regardless of cloud?